CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.1.1 (July 26, 2021)
+
+NOTES:
+
+* No new functianality, works with current sandbox
+
+* Fix GitHub actions and update used Go version [#19](https://github.com/tinkerbell/terraform-provider-tinkerbell/pull/19) 
+* Update Tinkerbell and linter, add linter and codespell to CI [#20](https://github.com/tinkerbell/terraform-provider-tinkerbell/pull/20)
+* Add/update DCO, CoC, contributing ... files [#24](https://github.com/tinkerbell/terraform-provider-tinkerbell/pull/24)
+* Update to latest sandbox version and remove some workarounds [#23](https://github.com/tinkerbell/terraform-provider-tinkerbell/pull/23)
```

Comment:
I wouldn't say latest, as IMO it indicates latest for the time of new release, which is not true, as it has been updated in Jaunary.
```suggestion
* Update to newer sandbox version and remove some workarounds [#23](https://github.com/tinkerbell/terraform-provider-tinkerbell/pull/23)
```